package.json

```diff
@@ -18,7 +18,8 @@
     "/npm-shrinkwrap.json",
     "/oclif.manifest.json",
     "/scripts/postinstall.js",
-    "/src/**/*.js"
+    "/src/**/*.js",
+    "!/src/**/*.test.js"
```

Comment:
We should not publish test files. Is this globbing expression correct?

Reply:
Good catch, that looks correct.